frontend/src/pages/RecipientRecord/pages/__tests__/GoalsObjectives.js

```diff
@@ -151,6 +154,8 @@ describe('Goals and Objectives', () => {
     // No Filters.
     const noFilterUrl = '/api/recipient/401/region/1/goals?sortBy=goalStatus&sortDir=asc&offset=0&limit=10';
     fetchMock.get(noFilterUrl, { count: 2, goalRows: noFilterGoals, statuses: defaultStatuses });
+
+    fetchMock.get('/api/communication-logs/region/1/recipient/401?sortBy=communicationDate&direction=desc&offset=0&limit=5&format=json&purpose.in[]=RTTAPA%20updates', { rows: [], count: 0 });
```

Comment:
Does it make sense to assert any data in the new table here?

Reply:
sure